NOTES-NONSTOP.md

```diff
@@ -0,0 +1,155 @@
+NOTES FOR THE HPE NONSTOP PLATFORM
+==============================
+
+Requirement details
+-------------------
+
+ In addition to the requirements and instructions listed
+ in [INSTALL.md](INSTALL.md), the following are required as well:
+
+  * The TNS/X platform supports hardware randomization. Specify
+    the `--with-rand-seed=rdcpu` option to the `./Configure` script. This
+    is recommended but not required. `egd` is supported at 3.0 but cannot be
+    used if FIPS is selected.
+  * The TNS/E platform does not support hardware randomization, so specify
+    the `--with-rand-seed=egd` option to the `./Configure` script.
+    
+
+About C99 compiler
+------------------
+
+ The c99 compiler is required for building OpenSSL from source. While c11
+ may work, it has not been broadly tested. c99 is the only compiler
+ prerequisite needed to build OpenSSL 3.0 on this platform. You should also
+ have the FLOSS package installed on your system. The ITUGLIB FLOSS package
+ is the only FLOSS variant that has been broadly tested.
+ 
+About Prefix and OpenSSLDir
+---------------------------
+
+ Because there are many potential builds that must co-exist on any given
+ NonStop node, managing the location of your build distribution is crucial.
+ Keep each destination separate and distinct. Mixing any mode described in
+ this document can cause application instability. The recommended approach
+ is to specify the OpenSSL version and threading model in your configuration
+ options, and keeping your memory and float options consistent, for example:
+ 
+ * For 1.1 `--prefix=/usr/local-ssl1.1 --openssldir=/usr/local-ssl1.1/ssl` 
+ * For 1.1 PUT `--prefix=/usr/local-ssl1.1_put --openssldir=/usr/local-ssl1.1_put/ssl`
+ * For 3.0 `--prefix=/usr/local-ssl3.0 --openssldir=/usr/local-ssl3.0/ssl` 
+ * For 3.0 PUT `--prefix=/usr/local-ssl3.0_put --openssldir=/usr/local-ssl3.0_put/ssl`
+
+ Use the `_RLD_LIB_PATH` environment variable in OSS to select the appropriate
+ directory containing `libcrypto.so` and `libssl.so`. In GUARDIAN, use the
+ `=_RLD_LIB_PATH` search define to locate the GUARDIAN subvolume where OpenSSL
+ is installed.
+
+Threading Models
```

Comment:
Would you mind to put this section before the previous one? It introduces some abbreviations used in the previous

Reply:
Done